spec/lib/salus/scanners/gosec_spec.rb

```diff
@@ -18,7 +18,7 @@
         expect(
           info[:stderr]
         ).to include(
-          'blank_repository' # debug information
```

Comment:
no packages found is a bit confusing to me, it doesn't necessarily tell me there is not code found, aka blank repo, can we stick with the blank repo description?

Reply:
This comes from the gosec error logs. I think blank_repository also came from the gosec logs.

What do you suggest here?

Reply:
This is also stated in the message. The stderr is just additional output from the scanner itself.